chatbot/views.py

```
@@ -0,0 +1,28 @@
+    "how are you": "I'm just a bot, but I'm doing great!",
+    "bye": "Goodbye! Have a great day!",
+    "default": "Sorry, I didn't understand that. Can you rephrase?"
+}
+
+# Function to get chatbot response
+def get_bot_response(user_input):
+    user_input = user_input.lower()
+    return responses.get(user_input, responses["default"])
+
+# API view to handle chat
+def chat_api(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `chat_api`

(ANN201)